x-pack/plugins/security_solution/public/common/components/navigation/solution_grouped_nav/solution_grouped_nav_panel.tsx

```diff
@@ -105,5 +116,61 @@ const SolutionGroupedNavPanelComponent: React.FC<SolutionGroupedNavPanelProps> =
     </>
   );
 };
+export const SolutionNavPanel = React.memo(SolutionNavPanelComponent);
+
+const SolutionNavPanelCategories: React.FC<SolutionNavPanelCategoriesProps> = ({
+  categories,
+  items,
+  onClose,
+}) => {
+  const itemsMap = new Map(items.map((item) => [item.id, item]));
+
+  return (
+    <>
+      {categories.map(({ label, linkIds }) => {
+        const links = linkIds.reduce<DefaultSideNavItem[]>((acc, linkId) => {
+          const link = itemsMap.get(linkId);
+          if (link) {
+            acc.push(link);
+          }
+          return acc;
+        }, []);
+
+        return (
+          <Fragment key={label}>
```

Comment:
nit: The key here is coming from i18n.translate, might be better to actually assign them a key prop in x-pack/plugins/security_solution/public/management/links.ts and use that prop?